repositories/OptionRepository.js

```diff
@@ -7,6 +7,19 @@ const mapping = { AnswerId: "answerId" };
 const fromDb = mapFields(mapping);
 const toDb = mapFields(invert(mapping));
 
+const checkForExistingExclusive = async answerId => {
+  const existingExclusive = await Option.findAll().where(
+    toDb({
+      answerId,
+      mutuallyExclusive: true,
+      isDeleted: false
+    })
+  );
+  if (!isEmpty(existingExclusive)) {
+    throw new Error("There is already a exclusive checkbox on this answer.");
```

Comment:
Typo.  This should read: `There is already an exclusive checkbox on this answer`